keda/templates/22-metrics-deployment.yaml

```diff
@@ -107,18 +107,14 @@ spec:
             {{- end }}
           args:
           - /usr/local/bin/keda-adapter
-          - --port={{ .Values.service.portHttpTarget }}
+          - --port={{ .Values.prometheus.metricServer.port }}
```

Comment:
I don't think this is intentional?

Reply:
Yes it is. We had 2 different prometheus server and both allowed to change the port, one using `portHttpTarget` and the other with `metricServer.port`. As I had to remove one of them, I thought that `metricServer.port` is more clear about what does than `service.portHttpTarget`, but I have to remove `service.portHttpTarget` from values and readme (I missed that thing)

Reply:
After reviewing the docs later on I realized we still have the default metrics there that are not KEDA specific so it's OK!